tool/compute_tag_analytics.py

```diff
@@ -12,6 +12,7 @@
 log = Logger(__name__)
 
 CONVERSATIONS_COLLECTION_KEY = 'nook_conversations'
+CONVERSATIONS_SHARDS_COLLECTION_KEY = 'nook_conversation_shards'
```

Comment:
I'd recommend CONVERSATION_SHARDS_COLLECTION_KEY (conversation is singular) to match the actual key

Reply:
Thanks for the catch, fixed!